janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/MapReduceIndexManagement.java

```diff
@@ -177,9 +178,9 @@ public MapReduceIndexManagement(JanusGraph g) {
         janusGraphMapReduceConfiguration.set(JanusGraphHadoopConfiguration.SCAN_JOB_CONFIG_ROOT,
                 GraphDatabaseConfiguration.class.getName() + "#JOB_NS");
         // Copy the StandardJanusGraph configuration under JanusGraphHadoopConfiguration.GRAPH_CONFIG_KEYS
-        org.apache.commons.configuration.Configuration localConfiguration = graph.getConfiguration().getLocalConfiguration();
-        localConfiguration.clearProperty(Graph.GRAPH);
-        copyInputKeys(hadoopConf, localConfiguration);
+        org.apache.commons.configuration.Configuration localc = graph.getConfiguration().getConfigurationAtOpen();
```

Comment:
In 0.2 this variable is called `localbc`, now it is renamed to `localc`. Is it OK?

Reply:
Do you mean it may cause merge problem later when people do merging from 0.2 to master again?
I thought it was a typo to have 'localbc', so I changed it to 'localc'.  Maybe it means 'local base configuration'.  But it is a trivial issue.

Reply:
I don't think that it will be a problem when merging from 0.2 into the master but the name isn't intuitively understandable for me. `localConfiguration` has more meaning for me than `localc` or `localbc`. But I agree that it isn't a big issue.